test/format/format.sh

```diff
@@ -94,6 +100,18 @@ while :; do
 	-c)
 		config="$2"
 		shift ; shift ;;
+	-D)
+		[[ "$2" == /* ]] || {
+		    echo "$name: -D argument must be an absolute path"
```

Comment:
Is it necessary to only allow an absolute path?

Reply:
No; fixed in 2da3291ff.